data/data/agent/files/etc/assisted/agent-installer.env.template

```diff
@@ -9,3 +9,4 @@
 #   
 NODE_ZERO_IP={{.NodeZeroIP}}
 RELEASE_IMAGE={{.ReleaseImage}}
+AGENT_TUI_LOG_PATH=/var/log/agent/agent-tui.log
```

Comment:
I'm not sure if this log will be captured by the `agent-gather` scripts, in case we could follow up in a subsequent PR